descriptors/ModuleDescriptor-template.json

```diff
@@ -184,7 +184,12 @@
             "usergroups.collection.get",
             "usergroups.item.get",
             "users.collection.get",
-            "users.item.get"
+            "users.item.get",
```

Comment:
are you sure you need to do any changes here? bulk-ops calls data-export-spring if I'm not mistaken

Reply:
@NikitaSedyx you're right, permissions were added on mod-bulk-operations side

Reply:
@NikitaSedyx , @siarhei-charniak 
permissions were added mistakenly.
It's fixed